src/index.js

```diff
@@ -3,14 +3,16 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { Provider } from 'react-redux';
 import injectTapEventPlugin from 'react-tap-event-plugin';
+/* eslint-disable */
```

Comment:
It throws an error for not having the history package in the dependencies. It comes with React-Router though, so I don't see a need to specify it